doc/source/using_config.rst

```diff
@@ -734,6 +736,10 @@ Attributes
   The client key is used to encrypt traffic when uploading traffic
   to the server.
 
+* ``access-token``
+
+  The path to a token for optional HTTP bearer authentication.
+
 Normally, only the ``server-cert`` is required to securely *download* data
 from remote cache services, while both the ``client-key`` and ``client-cert``
 is required to securely *upload* data to the server.
```

Comment:
Maybe mention access token here to (as an alternative to securely upload data.

Reply:
Added, and also fixed inaccuracies in the description of certificate options.